packages/delivery-react-native/test/delivery.test.ts

```diff
@@ -1,13 +1,33 @@
-/* global describe, it, expect */
+import Client from '@bugsnag/core/client'
+import delivery from '../'
+import { App, Device, User, Breadcrumb } from '@bugsnag/core'
 
-const Client = require('@bugsnag/core/client')
-const delivery = require('../')
+interface NativeClientEvent {
+  errors: Array<{
+    errorMessage: string
+  }>
+  severity: string
+  severityReason: {
+    type: string
+    [key: string]: any
+  }
+  unhandled: boolean
+  app: App
+  device: Device
+  threads: unknown[]
```

Comment:
`Thread` is defined in [`common.d.ts`](https://github.com/bugsnag/bugsnag-js/blob/9f74bf2157260cb2579b7caa95da0fb325a1f76e/packages/core/types/common.d.ts#L126-L133)

In fact, could you build `NativeClientEvent` this type by doing `Pick<EventWithInternals, 'errors' | 'severity' | 'severityReason' etc. >`?

Reply:
Yes this is much better. Slightly more complicated than I'd hoped but definitely an improvement.